src/resources/Change/SoleProprietorshipResource.ts

```diff
@@ -17,7 +17,7 @@ export const SoleProprietorshipResource: ResourceIF = {
     nameChangeOptions: [
       CorrectionTypes.CORRECT_NEW_NR
     ],
-    typeChangeInfo: 'You can not change the business type of a Sole Proprietorship / DBA. You must form a new ' +
+    typeChangeInfo: 'You cannot change the business type of a Sole Proprietorship / DBA. You must form a new ' +
```

Comment:
[cannot vs can not](https://www.merriam-webster.com/words-at-play/cannot-vs-can-not-is-there-a-difference#:~:text=This%20dictionary%20defines%20cannot%20as,I%20cannot%20see%20any%20pie.)